src/components/pages/doc/sidebar/item/item.jsx

```diff
@@ -4,77 +4,60 @@ import React, { useState } from 'react';
 
 import Link from 'components/shared/link';
 import { DOCS_BASE_PATH } from 'constants/docs';
-import ChevronRight from 'icons/chevron-right.inline.svg';
+import ChevronRight from 'icons/chevron-right-sm.inline.svg';
 
-import SubItem from './sub-item';
+const isActiveItem = (items, currentSlug) =>
+  !!items?.find(
+    ({ slug, items }) => slug === currentSlug || (items && isActiveItem(items, currentSlug))
+  );
 
 const Item = ({ title, slug, isStandalone, items, isOpenByDefault, currentSlug }) => {
-  const [isOpen, setIsOpen] = useState(isOpenByDefault);
+  const [isOpen, setIsOpen] = useState(isOpenByDefault || slug === currentSlug);
 
-  const handleClick = () => setIsOpen((isOpen) => !isOpen);
+  if (!isOpen && isActiveItem(items, currentSlug)) {
+    setIsOpen(true);
+  }
 
-  const docSlug = isStandalone ? `/${slug}` : `${DOCS_BASE_PATH}${slug}/`;
+  const handleClick = () => {
+    setIsOpen((isOpen) => !isOpen);
+  };
 
+  const docSlug = isStandalone ? `/${slug}` : `${DOCS_BASE_PATH}${slug}/`;
+  const Tag = slug ? Link : 'button';
   return (
-    <li>
-      {slug ? (
-        <Link
-          className={clsx('w-full py-2.5 pl-4 text-left !text-lg font-bold', {
-            'font-bold text-primary-2': currentSlug === slug,
-          })}
-          theme="black"
-          size="sm"
-          to={docSlug}
-        >
-          {title}
-        </Link>
-      ) : (
-        <button
-          className="flex w-full items-center py-2.5 text-left transition-colors duration-200 hover:text-primary-2"
-          type="button"
-          onClick={handleClick}
-        >
-          <ChevronRight
-            className={clsx(
-              'mr-2 shrink-0 transition-transform duration-150',
-              isOpen ? 'rotate-90' : 'rotate-0'
-            )}
-          />
-          <span className="text-lg font-bold leading-snug">{title}</span>
-        </button>
-      )}
+    <li className="flex flex-col">
+      <Tag
```

Comment:
`cursor-pointer` for link?

Reply:
Tag is either button or link, so I believe it has `cursor-pointer`